include/wx/generic/grid.h

```diff
@@ -2745,6 +2745,7 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledCanvas
 
     wxGridTypeRegistry*    m_typeRegistry;
 
+public:
```

Comment:
This is going to make the diff a bit messier (unless `--color-moved` is used), but I think these enums should be moved to the public (albeit "implementation only") part of the class instead of changing the access mode like this.

Alternatively, we could make `wxGridOperations` friend of `wxGrid`, I think this would solve the problem as well and it makes sense to allow access to these classes as they're logically part of `wxGrid` anyhow.

Reply:
Thanks. I did the change to declare wxGridOperations a friend class.